src/components/button/AposButton.stories.js

```diff
@@ -0,0 +1,65 @@
+import { storiesOf } from '@storybook/vue'
+
+import AposButton from './AposButton.vue'
+const mixin = {
+  data () {
+    return {
+      busy: false
+    }
+  },
+  methods: {
+    fakeBusy: function () {
+      this.busy = true
+      setTimeout(() => {
+        this.busy = false
+      }, 1000)
+    }
+  }
+}
+
+storiesOf('Button', module)
+  .add('Default', () => ({
+    components: { AposButton },
+    template: '<AposButton @click="fakeBusy" v-bind:busy="busy" label="Filter" />',
+    ...mixin
+  }))
+  .add('Default, Disabled', () => ({
+    components: { AposButton },
+    template: '<AposButton @click="fakeBusy" label="Filter" v-bind:disabled="true" />',
+    ...mixin
+  }))
+  .add('Default, Busy', () => ({
```

Comment:
Let's do this for primary and danger as well since the black spinner won't work there.

Reply:
Not sure what you mean?

Reply:
Adding variations for those color backgrounds.